src/strings/en-us.json

```
@@ -1565,12 +1578,14 @@
     "SubtitleVerticalPositionHelp": "Line number where text appears. Positive numbers indicate top down. Negative numbers indicate bottom up.",
     "SubtitleWhite": "White",
     "SubtitleYellow": "Yellow",
+    "Success": "Success",
     "Suggestions": "Suggestions",
     "Sunday": "Sunday",
     "SyncPlayAccessHelp": "The SyncPlay feature enables to sync playback with other devices. Select the level of access this user has to the SyncPlay.",
     "SyncPlayGroupDefaultTitle": "{0}'s group",
     "TabAccess": "Access",
     "TabAdvanced": "Advanced",
+    "TabBackups": "Backups",
```

```suggestion
    "HeaderBackups": "Backups",
```

This is really minor, but we don't use tabs anymore so we probably shouldn't use "Tab" for new translations. :wink: